spec/getDirectoryStructure.test.js

```diff
@@ -0,0 +1,13 @@
+var getDirectoryStructure = require('../lib/getDirectoryStructure');
```

Comment:
The CI is failing because this file doesn't exist.
You might want to stick in an empty file so things will at least 'compile'